### PR TITLE
added unit tests for cube_build

### PR DESCRIPTION
**Pull request by jemorrison**
additional unit tests for cube_build

### CODE REVIEW

File: jwst/cube_build/tests/test_cube_build_step.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+from astropy.io import fits
+from jwst.datamodels import IFUImageModel
+from jwst.cube_build import CubeBuildStep
+from jwst.cube_build.file_table import ErrorNoAssignWCS
+from jwst.cube_build.cube_build import ErrorNoChannels
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+
+
+@pytest.fixture(scope='module')
+def miri_cube_pars(tmpdir_factory):
```
Comment (mfixstsci) [inline jwst/cube_build/tests/test_cube_build_step.py]: Hey @jemorrison why did you create a the `cube_pars` reference file instead of using the configured file? Is there a `cube_pars` reference file?

Comment (jemorrison) [reply jwst/cube_build/tests/test_cube_build_step.py]: @mfixstsci did you mean why did I create it for this test ?
I created it for the test because I did not think it was appropriate to use CRDS in a unit test. 
So I just hacked up a dummy one (but I actually used some of the code that creates the real one)

Comment (jemorrison) [reply jwst/cube_build/tests/test_cube_build_step.py]: There is some complex logic in using the reference table and I thought I should check it in a unit test

Comment (mfixstsci) [reply jwst/cube_build/tests/test_cube_build_step.py]: Hey @jemorrison, okay great. I was just curious. I noticed that you do this for all of the instruments so I figured there was a good reason for it, thanks!

File: jwst/cube_build/tests/test_wcs.py [hunk 2]
```diff
@@ -173,15 +173,15 @@ def test_setup_wcs():
     footprint = (ra_min, ra_max, dec_min, dec_max, lambda_min, lambda_max)
 
     pars_cube = {
```
Comment (mfixstsci) [inline jwst/cube_build/tests/test_wcs.py]: Hey @jemorrison just to capture the reason for the change, why is this change necessary? Does the code behave differently if we use `None` or `0.0`?

Comment (mfixstsci) [reply jwst/cube_build/tests/test_wcs.py]: Is this because of the `assert` for `math.isclose`?

Comment (jemorrison) [reply jwst/cube_build/tests/test_wcs.py]: As I recall that was the reason -- to make assert for math.isclose work

**Comment by stscieisenhamer** [comment]
@mfixstsci Is this all ok now? If so, could you approve.

**Comment by mfixstsci** [comment]
Hey @stscieisenhamer everything looks good to me. I got all the answers I needed.
